src/wp-includes/class-wpdb.php

```diff
@@ -3950,14 +3779,12 @@ public function get_charset_collate() {
 	 * @since 2.7.0
 	 * @since 4.1.0 Added support for the 'utf8mb4' feature.
 	 * @since 4.6.0 Added support for the 'utf8mb4_520' feature.
-	 * @since 6.1.0 Added support for the 'identifier_placeholders' feature.
 	 *
 	 * @see wpdb::db_version()
 	 *
 	 * @param string $db_cap The feature to check for. Accepts 'collation', 'group_concat',
-	 *                       'subqueries', 'set_charset', 'utf8mb4', 'utf8mb4_520',
-	 *                       or 'identifier_placeholders'.
-	 * @return bool True when the database feature is supported, false otherwise.
+	 *                       'subqueries', 'set_charset', 'utf8mb4', or 'utf8mb4_520'.
+	 * @return bool True when the database feature is supported, false otherwise
```

Comment:
```suggestion
	 * @return bool True when the database feature is supported, false otherwise.
```

Reply:
Great catch @SergeyBiryukov. Resolved in https://github.com/WordPress/wordpress-develop/pull/3541/commits/2e71c8b8214f8c588e1e3d415174fff5c8851f9a.